src/__tests__/element-queries.js

```diff
@@ -66,4 +67,18 @@ test('totally empty label', () => {
   expect(() => getByLabelText('')).toThrowErrorMatchingSnapshot()
 })
 
+test('using jest helpers to assert element states', () => {
+  const {queryByTestId, getByTestId} = render(
+    <span data-testid="count-value">2</span>,
+  )
+
+  //other ways to assert your test cases, but you don't need all of them.
+  expect(queryByTestId('count-value')).toBeInTheDOM()
+  expect(queryByTestId('count-value')).toHaveTextContent('2')
+  expect(getByTestId('count-value')).toSatisfyDOM(el => el.textContent === '2')
```

Comment:
I'm starting to question the value of this matcher. Maybe instead we should just recommend people use [jest-extend](https://github.com/jest-community/jest-extended#tosatisfypredicate) instead. This is basically the same as `toSatisfy` from there.

The other matchers are still very valuable though!

Reply:
@kentcdodds `toSatisfyDOM` seems to be a fit for me in this library. Why we need to ask user to use jest-extend for a such a simple call? Yes, its almost the same code, but I feel it should be added. Anyways, let me know if you think otherwise.

Reply:
I agree that this matcher can be useful, but I'd much rather be very selective about what we include in this library. It's much easier to exclude something and add it later than add something and remove it later. So I lean on the side of excluding things. I'm thinking most people wont need this and those who do will be happy to include jest-extend I think.